Sources/linenoise/linenoise.swift

```diff
@@ -178,6 +178,7 @@ public class LineNoise {
     // MARK: - Terminal handling
     
     private static func isUnsupportedTerm(_ term: String) -> Bool {
+        guard ProcessInfo.processInfo.environment["XPC_SERVICE_NAME"] == nil else { return true }
```

Comment:
This should either check for the `XCODE_VERSION_ACTUAL` env var, _or_ check whether the XPC service name contains "Xcode" (it could be a different XPC). The former is probably better.
You might also want to wrap it in a `#if os(macOS)`, saves a tinsy bit of overhead on other platforms.

Reply:
`XCODE_VERSION_ACTUAL` seems not to be set here. Will take the other suggestions into account, thanks.